documentation/book/proc-changing-kafka-user.adoc

```diff
@@ -15,9 +15,15 @@ This procedure describes how to change the configuration of an existing Kafka us
 
 .Procedure
 
+You can edit the user either by preparing a YAML file containing the desired `KafkaUser` or by using {ProductPlatformName} tools.
```

Comment:
If this is an introduction it should be above the procedure - procedures should only be numbered steps.